.devcontainer/Dockerfile

```diff
@@ -0,0 +1,40 @@
+FROM golang:1.12
+
+# Avoid warnings by switching to noninteractive
+ENV DEBIAN_FRONTEND=noninteractive
```

Comment:
unset this env on the end of the build, prevents nasty surprises for users

Reply:
Nice catch! Addressed.